stglib/core/utils.py

```diff
@@ -744,6 +745,79 @@ def loadmat(filename):
     data = spio.loadmat(filename, struct_as_record=False, squeeze_me=True)
     return _check_keys(data)
 
+def read_samplingrates_continuous(ds,conn):
```

Comment:
Can you move these two new functions up above `def read_globalatts`? I like to keep the matlab and a few other general purpose functions at the end.

Reply:
done